src/bcf-toolkit/Utils/CollectionExtensions.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.ObjectModel;
+
+namespace BcfToolkit.Utils;
+
+public static class CollectionExtensions {
+  public static void AddIfNotExists<T>(this Collection<T> collection, T item) {
```

Comment:
This is specifically used for creating the extensions, where all the authors, commenters, users, and topic related enums are collected. The bcf uses collection instead of HashSet, so i have to check if the values which have to be added to the collection are not duplicated.